CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 ### 🔄 Changed
 - Dropped iOS 12 support [#3285](https://github.com/GetStream/stream-chat-swift/pull/3285)
 - Increase QoS for `Throttler` and `Debouncer` to `utility` [#3295](https://github.com/GetStream/stream-chat-swift/issues/3295)
+- Fetch database items immediately if the initial fetch has not finished in background database observers [#3304](https://github.com/GetStream/stream-chat-swift/issues/3304)
```

Comment:
It probably does not make sense to provide a changelog entry 🤔 This is just implementation details. What does the customer gain with this?

Reply:
Probably makes sense, all of this is internal details. I'll remove it.

Reply:
It can actually be useful - but re-worded in a way that now you can reliably access database items immediately.

Reply:
Tried to reword it

Reply:
Reworded it again. WDYT?

Reply:
The most generic version I could think of:
`Improve reliability of accessing database models`